source/MaterialXView/Viewer.cpp

```diff
@@ -1568,7 +1588,21 @@ void Viewer::computeCameraMatrices(mx::Matrix44& world,
     world *= mx::Matrix44::createTranslation(_modelTranslation).getTranspose();
 }
 
-void Viewer::updatePropertyEditor()
+void Viewer::updateDisplayedProperties()
 {
     _propertyEditor.updateContents(this);
+    mx::TypedElementPtr elem = getSelectedMaterial() ? getSelectedMaterial()->getElement() : nullptr;
+    const std::string& node = (elem && !elem->getAttribute("node").empty()) ? elem->getAttribute("node"): "";
```

Comment:
One minor note here:  If the return value of `elem->getAttribute("node")` is empty, then we can use that empty string as the value of this variable, saving us a bit of extra logic to explicitly check for the empty string and substitute `""`.